modules/local/make_bed_mask.nf

```diff
@@ -19,13 +19,22 @@ process MAKE_BED_MASK {
 
     input:
     tuple val(meta), path(vcf), path(bed)
+    path  fasta
 
     output:
-    tuple val(meta), path("*.bed"), emit: bed
+    tuple val(meta), path("*.bed")  , emit: bed
+    tuple val(meta), path("*.fasta"), emit: fasta
 
     script:  // This script is bundled with the pipeline, in nf-core/viralrecon/bin/
     def prefix = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
     """
-    make_bed_mask.py $vcf $bed ${prefix}.bed
+    make_bed_mask.py \\
+        $vcf \\
+        $bed \\
+        ${prefix}.bed
+
+    ## Rename fasta entry by sample name and not reference genome
```

Comment:
Wouldn't this be a little bit hidden here, I know is a small snippet, but maybe be worth renaming the module?

Reply:
Yeah, I agree it's a little dirty. The problem is that if we don't rename here then the fasta entries in the consensus sequence has the same name as the reference genome and not the actual sample name. This obviously isn't ideal for any tools that use the name in the consensus sequence for downstream analysis e.g. Pangolin. This is the only place we can put it really because the other modules are proper ones from nf-core/modules. I think it's fine to keep the module name as it is because renaming the consensus is kinda related to making the bed mask in the first place and is part of the same process.